examples/fluids/qfunctions/newtonian.h

```diff
@@ -783,4 +784,28 @@ CEED_QFUNCTION(PressureOutflow_Jacobian_Prim)(void *ctx, CeedInt Q,
   return PressureOutflow_Jacobian(ctx, Q, in, out, StateFromY, StateFromY_fwd);
 }
 
+// *****************************************************************************
+// This QFunction implements the Harten Lax VanLeer (HLL) approximate Riemann
+// solver. Taking in two states.
+// *****************************************************************************
+CEED_QFUNCTION_HELPER int Harten_Lax_VanLeer_Flux(void *ctx, CeedInt Q,
+    const CeedScalar *const *in, CeedScalar *const *out,
+    StateFromQi_t StateFromQi, StateFromQi_fwd_t StateFromQi_fwd) {
```

Comment:
These are just generic states.
```suggestion
    State left, State right, const CeedScalar normal[3]) {
```